packages/shared/scope-hoisting/src/formats/commonjs.js

```diff
@@ -0,0 +1,213 @@
+// @flow
```

Comment:
can these files be `// @flow strict-local`?

Reply:
Seems like they can't:
```
Importing from an untyped module makes it `any` and is not safe! Did you mean to add `// @flow` to the top of `@babel/types`? (`untyped-import`)
```

Reply:
yeah... I really wish we had flow types for Babel